config/crd/bases/tests.testkube.io_testsuites.yaml

```diff
@@ -356,6 +356,18 @@ spec:
                     description: test kubernetes namespace (\"testkube\" when not
                       set)
                     type: string
+                  runningContext:
```

Comment:
exclude this file from pr. it was done by purpose

Reply:
Discarded changes from file